packages/keyring/src/suri.spec.ts

```diff
@@ -5,10 +5,12 @@
 
 import { u8aToHex } from '@polkadot/util';
 import { cryptoWaitReady } from '@polkadot/util-crypto';
+import { KeypairType } from '@polkadot/util-crypto/types';
```

Comment:
```suggestion
import type { KeypairType } from '@polkadot/util-crypto/types';
```

Would just move it first where the types generally go.